app/api/helpers/db.py

```diff
@@ -82,6 +82,25 @@ def safe_query(model, column_name, value, parameter_name):
     )
 
 
+def safe_query_kwargs(model, kwargs, parameter_name):
+    """
+    :param model: db Model to be queried
+    :param kwargs: it contains parameter_name's value eg kwargs['event_id']
+                where parameter_name='event_id'
+    :param parameter_name: Name of parameter to be printed in json-api error
+                message eg 'event_id'
+    :return:
+    """
+    return safe_query_without_soft_deleted_entries(
```

Comment:
You are replacing `safe_query` method, so you should use it instead, not `safe_query_without_soft_deleted_entries`

And move it above ``safe_query_without_soft_deleted_entries